setup.py

```diff
@@ -46,11 +46,12 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
     ],
     keywords="python pagination",
     package_dir={"": "src"},
     packages=find_packages(where="src"),  # Required
     package_data={NAME: ["py.typed"]},
     python_requires=">=3.7, <3.10",
     install_requires=["SQLAlchemy", "pydantic"],
```

Comment:
this should also change to <=3.10

Reply:
Well I think it should work with 3.11 already, so maybe <=3.11?

Reply:
OK!